resources/frontend/vs_code.md

```diff
@@ -0,0 +1,22 @@
+## What is VS Code?
+VS Code, or Visual Studio Code, is a source code editor that is available for Windows, macOS, and Linux. VSCode comes with built-in support for JavaScript, TypeScript, and Jode.js.
+
+### How do you install VS Code on Mac?
+To install VS Code on Mac, first go to https://code.visualstudio.com/. On that page, there is a blue button on the left side of the page titled "Download for Mac." After clicking on the button, a zip file should appear. Click on the zip file to open VSCode in the "Downloads" folder on your computer.
+
+### What are some key features?
+VS Code offers syntax highlighting (with customizable colors), bracket-matching, and auto-indentation. It also includes an interactive debugger to inspect variables, look at source code, view call stacks, and execute commands in the console. In addtion, VS Code's integrated terminal makes committing and pushing files to Github go even smoother. 
```

Comment:
Make this section a list.
VSCode offers:
 - syntax highlighting
 - bracket-matching
...